ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/UploadDarResult.scala

```diff
@@ -10,6 +10,6 @@ object UploadDarResult {
   /** The package was successfully uploaded */
   final case object Ok extends UploadDarResult
 
-  /** The package was invalid for some reason */
```

Comment:
Now that we have gone for the synchronous pattern, I would fold the two case class hierarchies together. No need to keep both ...Reason and ...Result families. Let's just stick with the latter